ReSharper.FSharp/src/TypeProvidersLoader/TypeProvidersLoader.exe.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <runtime>
+      <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+        <dependentAssembly>
+          <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
+          <bindingRedirect oldVersion="2.0.0.0-9.9.9.9" newVersion="4.6.0.0"/>
```

Comment:
JFYI: we've migrated to `4.7.0.0` in `net201`, this file will need to be updated after rebase.

Reply:
Fixed